### DISCUSSION TITLE
为什么我在利用alist播放阿里云盘视频时c盘空间被不断占用？

### DISCUSSION THREAD

**Discussion by Faxmell** [AlistGo/alist#2725] [Q&A]
停止播放没几秒之后又恢复了，如果这是必须的，我c盘播着播着空间不够了有什么影响吗？

**Comment by 1249747637** [comment]
我也疑惑，我以为是要设置缓存文件的路径，结果发现temp_dir默认是在软件安装的路径下（我的是E盘），看视频的时候C盘写一次高峰，E盘没什么变化
